span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/redaction/PIIPCIField.java

```diff
@@ -0,0 +1,50 @@
+package org.hypertrace.core.spannormalizer.redaction;
+
+import static org.hypertrace.core.spannormalizer.constants.SpanNormalizerConstants.REDACTED_FIELD_PREFIX;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import org.hypertrace.core.datamodel.AttributeValue;
+
+public class PIIPCIField {
+
+  private String name;
+  private Set<String> tagKeySet;
+  private RegexInfo regexInfo;
```

Comment:
Better move `RegexInfo` as a public static class inside this class. This keeps related things close-by.

Reply:
Done. Have moved the enum inside this class too.